scale/README.md

```diff
@@ -0,0 +1,36 @@
+## Manual performance testing with the workload scale simulator
+
+This quickstart guide is for anyone looking to quickly test performance scenarios against a scaled cluster, e.g. manual testing of the ACS UI.
+
+To start a scale simulation, run the following from the root of the stackrox repo:
+
+```sh
+cd scale
+./launch_workload.sh <workload_name>
+# <workload_name> is the name of a yaml file in the `workloads` directory, without file extension
+# e.g. $ ./launch_workload.sh xlarge
+```
+
+**Note that this is a destructive operation on your current active `kubectx` cluster that cannot be easily undone.**
```

Comment:
I'd probably put the Note above the command

Reply:
Good call 👍